democracy/models/section.py

```diff
@@ -89,9 +89,7 @@ def plugin_implementation(self):
 
 
 class SectionImageManager(TranslatableManager, BaseModelManager):
-
-    def get_queryset(self):
-        return super(SectionImageManager, self).get_queryset().order_by('pk')
+    pass
```

Comment:
As commented below, should we still have `.distinct()` or something in the `get_queryset` here, if this is referred to in a section by the related name and the titles are repeated in several language? Or is it not needed?